rdmo/core/urls/v1/openapi.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib.auth.decorators import login_required
+from django.urls import path
+from django.views.generic.base import RedirectView
+
+from drf_spectacular.views import SpectacularAPIView, SpectacularRedocView, SpectacularSwaggerView
+
+app_name = 'v1-openapi'
+
+urlpatterns = [
+    path('schema/', login_required(SpectacularAPIView.as_view()), name='schema'),
+    path('swagger/', login_required(SpectacularSwaggerView.as_view(url_name='v1-openapi:schema')), name='swagger'),
+    path('redoc/', login_required(SpectacularRedocView.as_view(url_name='v1-openapi:schema')), name='redoc'),
+    path('', RedirectView.as_view(pattern_name='api', permanent=False))
```

Comment:
When I include in my rdmo-app
```py
urlpatterns += [
    path('api/v1/', include('rdmo.core.urls.v1.openapi', namespace='v1')),
]
```
I get an error for `http://localhost:8000/api/v1/`
> Reverse for 'api' not found. 'api' is not a valid view function or pattern name.

Where is this landing page supposed to be?

Reply:
```
from django.contrib import admin
from django.urls import include, path

from rdmo.core.views import about, api, home

urlpatterns = [
    path('', home, name='home'),
    path('about/', about, name='about'),
    path('api/', api, name='api'),

    path('', include('rdmo.core.urls')),
    path('api/v1/', include('rdmo.core.urls.v1')),
    path('api/v1/', include('rdmo.core.urls.v1.openapi')),

    path('admin/', admin.site.urls)
]
```